Install StorageOS v2 with Etcd (#31)
storageos/tutorials: install-storageos/000-introduction.md

This sandbox environment is a fully configured three node Kubernetes cluster.

In this tutorial, you will install StorageOS v2, create a StorageClass and use it to claim a persistent volume.

To install StorageOS we will proceed with the following steps:

1. Install etcd as a prerequisite

> 1. Install StorageOS Cluster Operator. Which is a Kubernetes native application

Reviewer: ```suggestion
1. Install the StorageOS Cluster Operator, a Kubernetes native application
```

   developed to deploy and configure StorageOS clusters, and assist with
   maintenance operations. 

> 1. Create a Secret to define the StorageOS credentials

Reviewer: ```suggestion
1. Create a Secret defining the StorageOS credentials
```

> 1. Create the StorageOS Cluster (Custom Resource)

Reviewer: ```suggestion
1. Create a StorageOS Cluster (Custom Resource)
```
